packages/workflow/src/RoutingNode.ts

```diff
@@ -108,24 +104,39 @@ export class RoutingNode {
 			abortSignal,
 		);
 
+		let credentialDescription: INodeCredentialDescription | undefined;
+
+		if (nodeType.description.credentials?.length) {
```

Comment:
Should an error be thrown when the credentials are not found?

Reply:
You cannot check if it is required, because the credential type has not been implemented in this case. An error is now thrown, because this error indicates that something is wrong in the node's code.